andela/ah-the-phoenix-frontend
# #163388096 A user should be able to reset their password

**What does this PR do?**
- Implement password reset feature

**Description of Task to be completed?**
- A user can get a link in their email to reset their password
- A user can be able to enter their new password

**How should this be manually tested?**
- Clone the github repository 
`git clone https://github.com/andela/ah-the-phoenix-frontend.git`
- Change your working directory to the project folder 
`cd ah-fulldeck-frontend`
- Checkout to the branch ft-password-reset-163388096
`git checkout ft-password-reset-163388096`
- Install the required dependencies using npm install
 -Run the local development server using:
 `npm start`
- Go to the link {{baseURL}}/reset-password/ and enter your email address
 - Check your email for the password reset link and click on it
 - Enter your new password and confirm it in the form provided


**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#163388096](https://www.pivotaltracker.com/story/show/163388096)

**Screenshots**
<img width="844" alt="Screen Shot 2019-03-14 at 11 47 25" src="https://user-images.githubusercontent.com/32046004/54343080-0ca4d300-464f-11e9-83d3-aba1c50ea1d4.png">
<img width="860" alt="Screen Shot 2019-03-14 at 11 47 06" src="https://user-images.githubusercontent.com/32046004/54343139-378f2700-464f-11e9-9372-b49a95b798e1.png">
